cfme/tests/services/test_service_catalogs.py

```diff
@@ -236,3 +238,57 @@ def test_order_service_after_deleting_provider(appliance, provider, setup_provid
     assert view.details.request_details.get_text_of("Request State") == "Finished"
     last_msg = f"Error: Source Template/Vm with id [{template_id}] has no EMS, unable to provision"
     assert view.details.request_details.get_text_of("Last Message") == last_msg
+
+
+@pytest.mark.tier(2)
+@pytest.mark.provider([InfraProvider], scope="module", selector=ONE)
+@pytest.mark.meta(automates=[1380197])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("import_data", [DatastoreImport("bz_1380197.zip", "bz_1380197", None)],
+                         ids=["datastore"])
+def test_service_catalog_email_request_approval_pending(request, appliance, import_datastore,
+                                                        generic_catalog_item):
+    """
+           Test to check is there have pending email approval status or not in log file after
+            ordering service catalog
+    Bugzilla:
+        1380197
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/6h
+        tags: service
+        testSteps:
+            1. order service catalog
+        expectedResults:
+            1. pending email approval status message should be there in automation.log file
+    """
+
+    service_catalogs = ServiceCatalogs(
+        appliance, catalog=generic_catalog_item.catalog, name=generic_catalog_item.name,
+    )
+
+    @request.addfinalizer
+    def _finalize():
+        # Remove Request
+        request_description = "Provisioning Service [{name}] from [{name}]".format(
+            name=generic_catalog_item.name
+        )
+        service_request = appliance.collections.requests.instantiate(
+            description=request_description, partial_check=True
+        )
+
+        service_request.remove_request(method="rest")
+
+    with LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[
+            (
+                r'INFO.*Invoking \[builtin\] method \[/\$/Object/send_email\] with '
+                r'inputs.*Service Request from  Pending Approval'
```

Comment:
This looks like its actually missing the from value? Is there a bug with CFME logging here?

Reply:
I reported this issue for some infra provider at first. They fixed only for Infra and I didn't bother to verify with some other by hand, but I think meanwhile hard-coded exception for cloud providers was added. I have no idea why the person putting it there did that.

BZ 1818172